.github/workflows/check-generated-modules.yml

```diff
@@ -0,0 +1,40 @@
+name: Check Generated Modules
+
+on:
+  schedule:
+    - cron:  '30 3 * * *'
+
+jobs:
+  check-generated-modules:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - "3.11"
+
+    steps:
+      - uses: actions/checkout@v4.1.0
+        with:
+          fetch-depth: 2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4.7.1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_scripts.txt
+      - name: Run scripts
+        run: |
+          python ./scripts/generate_multilevel_sensor_constants.py
+          python ./scripts/generate_notification_constants.py
+      - name: Check for changes
+        id: changes
+        uses: UnicornGlobal/has-changes-action@v1.0.12
+      - name: Create Pull Request
+        if: steps.changes.outputs.changed == 1
+        uses: peter-evans/create-pull-request@v5.0.2
+        with:
+          commit-message: "[create-pull-request] Update auto-generated constants"
+          delete-branch: true
+          title: "[create-pull-request] Update auto-generated constants"
```

Comment:
What does the `[create-pull-request]` mean? I'd remove that.

Reply:
it's the name of the action generating the PR. Do you have a suggestion for what to prepend it with? Just want to make it obvious that these are not user generated PRs

Reply:
Github actions should be the user that opens the PR. It's pretty clear that it's not user generated. We don't need to prepend anything to the PR title.

Example:
https://github.com/home-assistant-libs/pytradfri/pull/787

Reply:
ok fair enough